runtime/vm/j9vm.tdf

```diff
@@ -971,3 +971,5 @@ TraceEntry=Trc_VM_ComputeNPEMsgAtPC_Entry Overhead=1 Level=3 Template="ComputeNP
 TraceEvent=Trc_VM_ComputeNPEMsgAtPC_start Overhead=1 Level=3 Template="ComputeNPEMsgAtPC with bcCurrent : romClass (%p) romMethod (%p) temps (%p) bytecodeOffset (%p) bcCurrent (0x%x) npePC (0x%zx) npeFlag (%d) isMethodFlag (%d) npeMsg (%s)"
 TraceEvent=Trc_VM_ComputeNPEMsgAtPC_end Overhead=1 Level=3 Template="ComputeNPEMsgAtPC with bcCurrent : bcCurrent (0x%x) npePC (%u) npeFlag (%d) isMethodFlag (%d) npeMsg (%s)"
 TraceExit=Trc_VM_ComputeNPEMsgAtPC_Exit Overhead=1 Level=3 Template="ComputeNPEMsgAtPC : npePC (0x%zx) npeFlag (%d) isMethodFlag (%d) npeMsg (%s)"
+
+TraceEvent=Trc_VM_allocateClassLoader_SetFlag NoEnv Overhead=1 Level=1 Template="allocateClassLoader set flag J9CLASSLOADER_SHARED_CLASSES_ENABLED for class loader (%p)"
```

Comment:
Will we benefit from this being level 1? 3 or higher is preferred in production.

Reply:
Changed to level 3.